### ISSUE TITLE
Highlord Tirion Fordring

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5911]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Tirion inside ICC at the entrance behaves weirdly when pulled by trash mobs, he's not attacking them just running to them dealing no damage, after combat ends he doesn't reset back to his initial position but remains where the trash died AND the most annoying part is that sometimes he keeps the raid in combat bug if you are clearing the first trash pack with him forcing the raid to go out and back in or to alt f4 and log back to clear the combat.
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. pull trash to tirion
2. do some heals/dmg around and kill the trash
3. see how you are in combat bug (happens like 80% of the times he's being pulled)

**Include proofs for this behaviour**
![Screenshot_2](https://user-images.githubusercontent.com/36212275/87068690-a322dd80-c21e-11ea-82fe-17c877b40485.png)
see how he's not at his initial spot but remained where he got pulled last time
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, this will be fixed after the next scheduled maintenance.

Kind regards.
